lib/themes/theme.dart

```diff
@@ -101,6 +101,19 @@ class AppTheme {
   // Modal item spacer widths
   static const double modalIconSpacerWidth = iconContainerSize;
   static const double modalChevronSpacerWidth = spacingLarge;
+  static const double errorModalMargin = 16;
```

Comment:
all these consts here should be more universal at some point and reuse what exists instead of introducing new values - but for now that's fine, we will tackle this soon as part of unifying spacings/whitespace across the app